xapian-bindings/android/android.i

```diff
@@ -0,0 +1,401 @@
+%module(directors="1") xapian
```

Comment:
Since this is (apparently) identical to `xapian-bindings/java/java.i`, is there a way we can share the two rather than having a copy? It'll be easier to keep updated in future.